reproducibility_project/spe_subproject/src/engines/cassandra/example-project.py

```diff
@@ -0,0 +1,88 @@
+"""Setup for signac, signac-flow, signac-dashboard for this study."""
+import os
+import pathlib
+
+import flow
+import numpy as np
+from flow import environments
+
+
+class Project(flow.FlowProject):
+    """Subclass of FlowProject to provide custom methods and attributes."""
+
+    def __init__(self):
+        super().__init__()
+
+
+# ____________________________________________________________________________
+"""Setting progress label"""
+
+
+@Project.label
+@Project.pre(lambda j: j.sp.engine == "MYENGINENAME")
+def CreatedEngineInput(job):
+    """Check if the .json molecule topology was converted to engine input."""
+    return job.isfile("MYENGINEINPUTS")
+
+
+@Project.label
+@Project.pre(lambda j: j.sp.engine == "MYENGINENAME")
+def OutputThermoData(job):
+    """Check if the engine loaded the input files and wrote out thermo data."""
+    return job.isfile("NAMEOFMYTHERMOOUTPUT")
+
+
+@Project.label
+@Project.pre(lambda j: j.sp.engine == "MYENGINENAME")
+def FinishedSPECalc(job):
+    """Check if the log-spe.txt has been created."""
+    return job.isfile("log-spe.txt")
+
+
+# _____________________________________________________________________
+"""Setting up workflow operation"""
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "MYENGINENAME")
+@Project.post(CreatedEngineInput)
+@flow.with_job
+def LoadSystemSnapShot(job):
+    """Create initial configurations of the system statepoint."""
+    import mbuild as mb
+
+    molecule = job.sp.molecule
+    snapshot_directory = (
+        pathlib.Path(os.getcwd()).parents[1] / "src" / "system_snapshots"
```

Comment:
To make this a bit less brittle and relying on `os.getcwd`, lets use the signac project's methods to help us out.
https://docs.signac.io/projects/core/en/latest/api.html#signac.Project.root_directory

```suggestion
        pathlib.Path(Project.root_directory()/ "src" / "system_snapshots"
```

Not sure if this is exactly the syntax needed, but this should work with some tweaking. You might have to init the project first

```suggestion
        pr = Project()
        pathlib.Path(pr.root_directory()/ "src" / "system_snapshots"
```

Reply:
Great suggestion, thanks @justinGilmer. Added in [0c50067](https://github.com/mosdef-hub/reproducibility_study/pull/166/commits/0c5006779751c5c595be7c4d57b8f2647b9bf2a2)